lovesay/love.py

```diff
@@ -22,22 +22,8 @@ def get_file_path():
     filePath = f"{home}/.config/lovesay/quotes"
     
     return filePath
-
-def get_max_width():
-    
-    '''
-    get_file_path() -> int
-
-    Returns the current width of the terminal window
-
-    :params - None
-    '''
-    
-    cols, rows = shutil.get_terminal_size()
-
-    return cols
     
-def generate_quote(from_file = True, file_path=get_file_path(), quote=""):
+def generate_quote(max_width, from_file=True, file_path=get_file_path(), quote="", ):
```

Comment:
This is really minor, but most other functions that take `max_width` as an argument usually have it as the last arg. Moving it over for the sake of consistency would be wonderful :)

Reply:
Default arguments can't be followed by non-default arguments, so it would need a default value to go at the end, which feels like an awkward place to put a default value. 
```
  File "/usr/local/lib/python3.9/site-packages/lovesay/love.py", line 26
    def generate_quote(from_file=True, file_path=get_file_path(), quote="", max_width):
                                                                                     ^
SyntaxError: non-default argument follows default argument
```

Reply:
Completely forgot this was the case. Since it was just an issue of aesthetics and consistency anyways, I think keeping it as the first arg makes sense!